packages/design-tokens/docs/components/constants.tsx

```diff
@@ -105,7 +107,7 @@ const FONT_SIZE = reduce(
         name: "Name",
         transform: getTokenNameFromTuple(prefix),
       },
-      { name: "Pixels", transform: getTokenComment },
+      { name: "Weight", transform: getTokenValue },
```

Comment:
Shouldn't we use `Weight` only for font-weight? 

![image](https://github.com/user-attachments/assets/f6466723-bc0b-4d4c-8453-51e804c6176b)

Reply:
I'm removing this column since it doesn't make sense on font size!